t/conn.t

```diff
@@ -235,3 +235,52 @@ committed: true
 --- no_error_log
 [error]
 [lua]
+
+
```

Comment:
We are missing a line jump here. You should use the `reindex` script of the devel-utils repository.